addons/website_sale_stock/static/src/js/website_sale_reorder.js

```diff
@@ -25,10 +25,26 @@ patch(ReorderDialog.prototype, {
 
     stockCheckCombinationInfo(product) {
         // Products that should have a max quantity available should be limited by default.
-        if (product.combinationInfo.allow_out_of_stock_order || ! product.is_storable) {
+        // For normal products, skip if they allow out-of-stock orders or are non-storable.
```

Comment:
The definition of "normal" is a bit vague IMO :sweat_smile: 
```suggestion
        // For non-combo products, skip if they allow out-of-stock orders or are non-storable.
```